packages/EasyDoctrine/src/Timestampable/TimestampableImmutableTrait.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyDoctrine\Timestampable;
+
+use Carbon\CarbonImmutable;
```

Comment:
You have to add Carbon to `composer.json` this way:
```
"nesbot/carbon": "^1.39 || ^2.22"
```

Reply:
done